### DISCUSSION TITLE
Reliability of vector mbtiles maps

### DISCUSSION THREAD

**Discussion by demiantres** [mapsforge/vtm#882] [General]
Hello,

While playing around with mbtiles vector files from different sources (for instance from https://extract.bbbike.org/, https://github.com/klokantech/vector-tiles-sample) I noticed that some do not work at all or create faulty results (for instance disappearing or wrong clipping areas). On the other hand maps from OpenMapTiles (maptiler.com) work fine. Are mbtiles only partly supported in VTM or is this a problem of the map sources?

**Comment by devemux86** [comment]
This was contributed by the community for only Android and may not work with scoped storage.

You could check the related pull requests #708 and #740 and try contacting their authors.
